manual/src/ornate/cookbook.md

```diff
@@ -313,15 +313,48 @@ Additionally, you need to update any webpack plugins your config uses, to Webpac
 
 Webpack 4 has the potential to substantially reduce your webpack compilation times (80% reductions have been observed but your mileage may vary)
 
-## How to use get a list of assets
+## How to get and use a list of assets
 
 `scalajs-bundler` (version 0.13.0 onwards) will export a list of all assets produced by webpack. You can read that list on sbt
 
 ~~~ scala
 val files = (webpack in (Compile, fullOptJS)).value
 ~~~
 
-You can use it e.g. with `[sbt-native-packager](https://github.com/sbt/sbt-native-packager)` to add mappings as:
+You can use it e.g. with [sbt-native-packager](https://github.com/sbt/sbt-native-packager)` to add mappings as:
 ~~~ scala
-mappings in (Compile, packageBin) ++= (webpack in (Compile, fullOptJS)).value.map { f => f.data -> f.data.getName() },
+Universal/mappings ++= (Compile/fullOptJS/webpack).value.map{ f =>
+  f.data -> s"assets/${f.data.getName()}"
+},
 ~~~
+
+This will add all artifacts produced by the fully optimized Scala.JS run to the target archive. If you need to package additional libraries that have been aquired by `scalajs-bundler`, you can do something like:
+~~~
+    Universal/mappings ++= (Compile/fullOptJS/webpack).value.map{ f =>
```

Comment:
i guess this notation is sbt-1.1.x only right?

Reply:
Yes, I was using the new unified notation, forgot to mention that. I also haven't tested the approach with 0.13.x